test/e2e/flowaggregator_test.go

```diff
@@ -162,11 +162,75 @@ type testFlow struct {
 	dstPodName string
 }
 
+func TestFlowAggregatorSecureConnection(t *testing.T) {
+	skipIfNotFlowVisibilityTest(t)
+	skipIfHasWindowsNodes(t)
+	testCases := []struct {
+		testOption flowVisibilityTestOptions
```

Comment:
I think you could embed this type, but if you want to stick to a named field, rename `testOption` to `options`

Reply:
Updated. Thanks.